_includes/timeline.html

```diff
@@ -127,6 +127,19 @@
                             </div>
                         </li>
                         <li class="timeline-inverted">
+                            <div>
+                                <a href="/on-demand/graphql"><img class="timeline-image" src="./images/graphql-logo.svg" alt=""></a>
+                            </div>
+                            <div class="timeline-panel">
+                                <div class="timeline-heading">
+                                    <a href="/on-demand/graphql"><h2>Introduction to GraphQL</h2></a>
+                                </div>
+                                <div class="timeline-body">
+                                    <p class="text-muted">Learn how to configure your local git environment to work with GitHub and how to complete GitHub flow using the GitHub Extension for Visual Studio.</p>
```

Comment:
P.S. I realized after making these comments that mentioning one person would be weird because this has been a team effort, so @brianamarie I apologize for pinging you here - it really should have been a group ping.